db/queries.js

```diff
@@ -0,0 +1,201 @@
+const moment = require('moment');
+const Promise = require('bluebird');
+const knex = require('./knex');
+
+// References to Postgres tables
+const pairs = () => knex('pairs');
+const ticks = () => knex('ticks');
+const s5bars = () => knex('s5bars');
+
+// General Helper Functions
+const isValidDateTime = (timestamp) => {
+  return moment(timestamp, moment.ISO_8601, true).isValid();
+};
+
+
+/**
+ * Pairs Table Helper Functions
+ */
+const getAllPairs = () => {
+  return pairs().select();
+};
+
+const getPair = ({ id, name }) => {
+  if (id !== undefined) {
```

Comment:
There are a few of these in this file... do you need to strictly check against undefined or can you get away with if (!id) ?